Ginger/GingerCore/Actions/ActScript.cs

```diff
@@ -250,38 +302,28 @@ private void parseRC(string sRC)
                 }
             }
 
-            if (i>=0 && i2> 0)
```

Comment:
We need to keep it because currently it reads only output between RC start and end